### PR TITLE
Update cli.py

### PR DESCRIPTION
**Pull request by sebastianLedzianowski**
argNorm accepts input regardless of case (on the command line)

### CODE REVIEW

File: argnorm/cli.py [hunk 1]
```diff
@@ -9,15 +9,18 @@ def main():
...
-    parser.add_argument('--db', type=str,
+    parser.add_argument('--db', type=str.lower,
                         choices=['sarg', 'ncbi', 'resfinder', 'deeparg', 'megares', 'argannot', 'resfinderfg'],
                         help='The database you used to do ARG annotation.')
-    parser.add_argument('--hamronized', action='store_true', help='Use this if the input is hamronized (processed using the hAMRonization tool)')
-    parser.add_argument('-i', '--input', type=str, help='The annotation result you have')
-    parser.add_argument('-o', '--output', type=str, help='The file to save normalization results')
+    parser.add_argument('--hamronized', action='store_true',
+                        help='Use this if the input is hamronized (processed using the hAMRonization tool)')
+    parser.add_argument('-i', '--input', type=str.lower,
```
Comment (luispedro) [inline argnorm/cli.py:20]: This should not be `str.lower`, though. It's a filename and should be passed as is, no?

File: argnorm/cli.py [hunk 2]
```diff
@@ -9,15 +9,18 @@ def main():
...
+                        help='The annotation result you have')
+    parser.add_argument('-o', '--output', type=str.lower,
```
Comment (luispedro) [inline argnorm/cli.py:22]: Same as above

**Review by luispedro — CHANGES_REQUESTED** [review]
See the comments about `str.lower`

Also, please add some tests inside `integration_tests/run_argnorm.py`

**Comment by luispedro** [comment]
Thanks a lot for the effort. I actually was not aware that you could pass `str.lower` to `argparse`, that's pretty nice. But I think it should not apply to the `-i`/`-o` arguments

**Comment by sebastianLedzianowski** [comment]
Edited as directed. I hope everything is fine now.

File: integration_tests/run_argnorm.py [hunk 3]
```diff
@@ -26,13 +26,15 @@ def run_cli_test(tool, file, folder, db=None):
...
     run_cli_test('argsoap', 'args-oap.sarg.reads.tsv', folder)
+    run_cli_test('DEEParg', 'deeparg.deeparg.orfs.tsv', folder)
     run_cli_test('deeparg', 'deeparg.deeparg.orfs.tsv', folder)
     run_cli_test('amrfinderplus', 'amrfinderplus.ncbi.orfs.tsv', folder)
     run_cli_test('resfinder', 'resfinder.resfinder.reads.tsv', folder)
     run_cli_test('resfinder', 'resfinder.resfinder.orfs.tsv', folder)
 
-for db in ['argannot', 'megares', 'ncbi', 'resfinder', 'resfinderfg']:
+for db in ['ARGANNOT', 'argannot', 'MEGAres', 'megares', 'ncbi', 'resfinder', 'resfinderfg']:
     file = f'abricate.{db}.tsv'
```
Comment (luispedro) [inline integration_tests/run_argnorm.py:38]: This should now be

```python
    file  = f'abricate.{db.lower()}.tsv'
```

**Review by luispedro — CHANGES_REQUESTED** [review]
Tests are failing, but I think it's a trivial thing (see line comment).

Other than that, it could be merged!

**Comment by luispedro** [comment]
Thanks! Merged. This is much appreciated!

@Vedanth-Ramji : can you make a note in the `CHANGELOG` and docs about this change and credit @sebastianLedzianowski ?

**Pull request merged**